ProxyExtension/SplitTunnelDNSProxyProvider/SplitTunnelDNSProxyProvider.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import NetworkExtension
+
+final class SplitTunnelDNSProxyProvider : NEDNSProxyProvider {
```

Comment:
curious, do users get another pop-up asking for "DNS proxy" control or something?

Reply:
Yes, same popup that ask you to "Allow VPN...". Activating the extension works for both since 1 extension can hold n proxyManagers

Reply:
oh,i misunderstood i thought we only needed one pop-up and it would apply to both - but we DO need two sep pop-ups?

Reply:
yeah sorry, we need one popup per manager, and we have two managers: one for traffic and one for DNS requests (you will see two separate entries in Filters & Proxies section).
The one for activating the extension remains the same, since 1 extension can hold both managers